### ISSUE TITLE
service catalog - multiple tickets of a form answer does not show

### ISSUE THREAD

**Issue by btry** [pluginsGLPI/formcreator#759]
* Create a form with 2 target tickets
* answer the form
* use a user without any group membership to show the issues in the service catalog (profile with simplified interface, formcreator's service catalog enabled)

When the user clicks in the issue and views the tickets of  the issue, he gets "no tickets found"

![image](https://user-images.githubusercontent.com/14139801/31053761-e9a879c4-a6a4-11e7-9420-6217dddfa485.png)

sql-errors.log contains 
```
2017-10-01 12:13:34 [2@workstation]
  *** MySQL query error:
  SQL: SELECT  DISTINCT `glpi_tickets`.*,
                        `glpi_itilcategories`.`completename` AS catname
                        , `glpi_entities`.`completename` AS entityname,
                       `glpi_tickets`.`entities_id` AS entityID 
                FROM `glpi_tickets`  LEFT JOIN `glpi_groups_tickets`
                  ON (`glpi_tickets`.`id` = `glpi_groups_tickets`.`tickets_id`)
               LEFT JOIN `glpi_tickets_users`
                  ON (`glpi_tickets`.`id` = `glpi_tickets_users`.`tickets_id`)
               LEFT JOIN `glpi_suppliers_tickets`
                  ON (`glpi_tickets`.`id` = `glpi_suppliers_tickets`.`tickets_id`)
               LEFT JOIN `glpi_itilcategories`
                  ON (`glpi_tickets`.`itilcategories_id` = `glpi_itilcategories`.`id`)
               LEFT JOIN `glpi_tickettasks`
                  ON (`glpi_tickets`.`id` = `glpi_tickettasks`.`tickets_id`)
               LEFT JOIN `glpi_items_tickets`
                  ON (`glpi_tickets`.`id` = `glpi_items_tickets`.`tickets_id`)
                LEFT JOIN `glpi_entities`
                        ON (`glpi_entities`.`id` = `glpi_tickets`.`entities_id`) 
                WHERE (`glpi_items_tickets`.`items_id` = \'109\'  AND `glpi_items_tickets`.`itemtype` = \'PluginFormcreatorForm_Answer\') AND (`glpi_tickets`.`users_id_recipient` = \'2\'
                                   OR (`glpi_tickets_users`.`tickets_id` = `glpi_tickets`.`id`
                                       AND `glpi_tickets_users`.`users_id`
                                            = \'2\')
                                   OR `glpi_groups_tickets`.`groups_id` IN ()) 
                AND glpi_tickets.is_deleted = 0
                ORDER BY `glpi_tickets`.`date_mod` DESC
                LIMIT 20
  Error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ')) 
                AND glpi_tickets.is_deleted = 0
                ORDER BY `gl' at line 23
  Backtrace :
  inc/ticket.class.php:6020                          
  inc/ticket.class.php:882                           Ticket::showListForItem()
  inc/commonglpi.class.php:482                       Ticket::displayTabContentForItem()
  ajax/common.tabs.php:96                            CommonGLPI::displayStandardTab()
```

hightlight
```
OR `glpi_groups_tickets`.`groups_id` IN ()) 
```

https://github.com/glpi-project/glpi/blob/9.2/bugfixes/inc/ticket.class.php#L5994

In the code, the SQL may be generated from an empty array $_SESSION['glpigroups'].

**Comment by btry** [comment]
adding a group to the user viewing the details of the issue solves the SQL error and the tickets show correctly

![image](https://user-images.githubusercontent.com/14139801/31053809-be9eae3c-a6a5-11e7-9200-548033c426f4.png)

**Comment by btry** [comment]
fix for GLPI approved, should be available in 9.2.1

**Comment by btry** [comment]
fix for GLPI merged. the issue will disappear with GLPI 9.2.1
